lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -35,12 +35,28 @@ under the License.
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.IO.Compression.FileSystem" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="API\ClientFactory.cs" />
+    <Compile Include="API\Exceptions\ClasspathException.cs" />
+    <Compile Include="API\Exceptions\JavaNotFoundException.cs" />
+    <Compile Include="API\IREEFClient.cs" />
+    <Compile Include="API\JobSubmission.cs" />
     <Compile Include="CLRBridgeClient.cs" />
+    <Compile Include="Common\ClientConstants.cs" />
+    <Compile Include="Common\DriverFolderPreparationHelper.cs" />
+    <Compile Include="Common\FileSets.cs" />
+    <Compile Include="Common\JavaClientLauncher.cs" />
+    <Compile Include="Local\LocalClient.cs" />
+    <Compile Include="Local\LocalRuntimeClientConfiguration.cs" />
+    <Compile Include="Local\Parameters\LocalRuntimeDirectory.cs" />
+    <Compile Include="Local\Parameters\NumberOfEvaluators.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="packages.config" />
     <None Include="run.cmd" />
```

Comment:
I don't want to break the old client just yet. The new one doesn't need it and we should remove it once we moved all code over to use that one.